src/main/java/com/bitso/BitsoOperation.java

```diff
@@ -36,13 +36,69 @@ public String toString() {
         return Helpers.fieldPrinter(this);
     }
 
-    private class BalanceUpdate{
+    public String getEntryId() {
+		return entryId;
+	}
+
+	public void setEntryId(String entryId) {
+		this.entryId = entryId;
+	}
+
+	public String getOperationDescription() {
+		return operationDescription;
+	}
+
+	public void setOperationDescription(String operationDescription) {
+		this.operationDescription = operationDescription;
+	}
+
+	public Date getOperationDate() {
+		return operationDate;
+	}
+
+	public void setOperationDate(Date operationDate) {
+		this.operationDate = operationDate;
+	}
+
+	public BalanceUpdate[] getAfterOperationBalances() {
+		return afterOperationBalances;
+	}
+
+	public void setAfterOperationBalances(BalanceUpdate[] afterOperationBalances) {
+		this.afterOperationBalances = afterOperationBalances;
+	}
+
+	public JSONObject getDetails() {
+		return details;
+	}
+
+	public void setDetails(JSONObject details) {
+		this.details = details;
+	}
+
+	public class BalanceUpdate{
         String currency;
         BigDecimal amount;
 
         public BalanceUpdate(JSONObject o) {
             this.currency = Helpers.getString(o, "currency");
             this.amount = Helpers.getBD(o, "amount");
         }
+
+		public String getCurrency() {
+			return currency;
+		}
+
+		public void setCurrency(String currency) {
+			this.currency = currency;
+		}
+
+		public BigDecimal getAmount() {
+			return amount;
+		}
+
+		public void setAmount(BigDecimal amount) {
+			this.amount = amount;
+		}
```

Comment:
Same here. You've added tabs into the mix.

Please download the Java Formatter from the wiki and adhere to that :)